oxen/tests/test_remote_repo.py

```
@@ -54,6 +54,20 @@
     assert not remote_repo.file_exists(file_path, "main")
 
 
+def test_remote_repo_branch_exists(
+    celeba_remote_repo_one_image_pushed: Tuple[Repo, RemoteRepo], shared_datadir
+):
+    _local_repo, remote_repo = celeba_remote_repo_one_image_pushed
+    file_path = "examples.tsv"
+    relative_path = os.path.join("ChatBot", file_path)
+    full_path = os.path.join(shared_datadir, relative_path)
```

Ruff (0.8.2)

Local variable `full_path` is assigned to but never used

Remove assignment to unused variable `full_path`

(F841)